src/linux/helpers/nolibc-tweaks.patch

```diff
@@ -5,7 +5,7 @@ index f68baf8f3..18710dd30 100644
 @@ -181,6 +181,8 @@ struct sys_stat_struct {
  	_arg1;                                                                \
  })
- 
+
```

Comment:
Most of the changes to this file look like a text editor blindly took out all trailing whitespace. Please revert, so we can keep the history clean.